### PR TITLE
bump guest-components + ITA kustomization

### PR DESCRIPTION
**Pull request by mythi**

### CODE REVIEW

File: kbs/config/kubernetes/ita/kustomization.yaml
Comment (fidencio) [inline kbs/config/kubernetes/ita/kustomization.yaml:7]: Just a note fur the future, but this doesn't block this PR.

I've asked @mythi about having the ITA image set here by default, but it seems that even if we have that the kustomization is not picking up the correct image.

We'll solve this and add the image here as a follow-up.

**Review by fidencio — APPROVED** [review]
lgtm, thanks @mythi!

**Review by jodh-intel — APPROVED** [review]
Thanks, @mythi.

lgtm

**Comment by Xynnn007** [comment]
Thanks. Do we need to firstly bump gc and then change the rev of trustee to use that one?

**Comment by fidencio** [comment]
> Thanks. Do we need to firstly bump gc and then change the rev of trustee to use that one?

We're using the guest components hash of the latest commit, which includes the changes that we need, so I think we're good to go here @Xynnn007.

My next step will do bump everything on the Kata Containers side as soon as this one is merged.

**Review by Xynnn007 — APPROVED** [review]

**Pull request merged**
